pathod/utils.py

```diff
@@ -57,7 +59,7 @@ def inner_repr(s):
         Returns the inner portion of a string or unicode repr (i.e. without the
         quotes)
     """
-    if isinstance(s, unicode):
+    if six.PY2 and isinstance(s, unicode):
```

Comment:
Assuming this function accepts both bytes and unicode, this case should also cover the "we supplied bytes here in py3" case, which prefixes with `b`.